File: Java_Programs/While10.java
//wap to accept a no and then print whether it is prime no. or not.
import java.util.*;
class While10
{
	public static void main(String []args)
	{
		int num,i=1,count=0;

		Scanner s=new Scanner(System.in);

		System.out.println("Enter a number: ");
		num = s.nextInt();

		while(i<=num)
		{
			if(num % i == 0){
				count++;
			}
			if(count==2){
				System.out.println("It is prime number");
			}
			else {
				System.out.println("It is not prime number");
			}
		}
	}
}

File: C_Program/while10.c
//wap to accept a nos and then print whether it is prime no. or not.

#include<stdio.h>
int main()
{
	int num,i,count=0;
	printf("\n enter the no.: ");
	scanf("%d",&num);
	for(i=1;i<=num;i++)
	{
		if(num%i==0)
		{
			count++;
		}
	}
	if(count==2)
	{
		printf("\n It is prime no.");
	}
	else
	{
		printf("\n It is not prime no.");
	}
}